test/tests/restart/kernel_restartable/tests

```diff
@@ -29,6 +30,7 @@
       input = 'kernel_restartable_custom_name.i'
       exodiff = 'kernel_restartable_custom_name_out.e'
       detail = "to name problem objects in a simulation and"
+      recover = false
```

Comment:
Why?

Reply:
Restart is already tested below in `[custom_name2]`.